script.js

```diff
@@ -93,3 +95,10 @@ allBtns.addEventListener("click", function (e) {
     timer();
   }
 });
+
+
+toggle.addEventListener("click", () => {
+  container.classList.toggle("dark")
+    ? (toggle.firstElementChild.className = "far fa-moon")
+    : (toggle.firstElementChild.className = "far fa-sun");
+});
```

Comment:
@Farhan-Ali2002 This is very good, but can you add some more code that whenever the stopwatch starts the "toggle div" to be removed so that the animations can start, and make the "toggle div" not clickable with "pointer-events: none". If you have problem with implementing any code that I said please feel free to tell me, I will help you with it or I will approve your changes and fix the small stuff on my own :).

Reply:
yeah I will definitely make the changes:) The thing is i have never worked with sass yet so if you have any good sources through which i can get to learn about it, please do share! Also it will be very helpful if you explain me the javascript part you talking about :))

Reply:
Like for now I have copy pasted my CSS to the SCSS file and replaced the colors with their variable names. tell me is that it or is there more to it?

Reply:
@Farhan-Ali2002 It's fine if you haven't worked with it before, do you know how to use the terminal?

Reply:
oh yes! I have "npm install" the node-sass module

Reply:
@Farhan-Ali2002 okay after u install it, write "npm run compile-sass" this will compile everything you do at the scss file to the css file and then go to a live server to see the changes apply from scss file directly to it.

Reply:
well I just made the changes and it works. Don't Know the code is efficient or not but it works though :DD

Reply:
@Farhan-Ali2002 Everything seems to be working perfectly, well done. :)

Reply:
Thankyou @abitsalihu :)) it was fun. Learned some new stuff!

Reply:
Also @abitsalihu I am new to the open source so if you could share some tips on how can i contribute more to it. it will be really helpful:)

Reply:
@Farhan-Ali2002 I'm new too, I started 15 days ago lol. 

How I find projects to contribute to is by going to the issue section and on the search bar I look for projects by : is:open is:issue language:css 

Or I look for projects by searching: is:open is:issue label:"good first issue" 

With these searches you find projects which are more beginner friendly.

Reply:
oh! Thanks then:DD